raiden/network/transport/matrix.py

```diff
@@ -674,12 +672,9 @@ def send_delivered_for(message: SignedMessage):
             self.log.warning('Exception while processing message', exc_info=True)
             return
 
-    def _send_queued_messages(self):
-        for queue_identifier, events in self._queueids_to_queues.items():
-            node_address = self._raiden_service.address
-            for event in events:
-                message = _event_to_message(event, node_address)
-                self._raiden_service.sign(message)
+    def _send_queued_messages(self, queueids_to_queues):
```

Comment:
You need to adapt all uses of `_queueids_to_queues` to iterate over `Message`s instead of `SendMessageEvent`s

Reply:
I don't follow, what are you referring to?

Reply:
The methods I enumerated above use `_queueids_to_queues` as the values being `List[SendMessageEvent]`, specifically iterating over the events and comparing the `message_identifier` of the message with the attribute of the same name of the event. If now the `QueueIdsToQueues` is an alias for `Dict[QueueIdentifier, List[Message]]`, you need to update the type alias and ensure these places where this mapping is used comply with this new signature. If the `Message` instance passed to transport is the same inside the various _versions_/_instances_ of this map, or all `Message` instances implements correctly equality comparison, you may want to optimize these lookups by using something like `message in self._queueids_to_queues[queue_identifier]`, instead of the `any` iterator.